pages/your_tasks/ethics.md

```diff
@@ -0,0 +1,116 @@
+        * Automated sharing of research data after a specific period or milestones and deliverables in a project can be a good mechanism to enhance the openness of a project ([data management coordination](dm_coordination))
+​
+
+## How can I identify regulations, guidelines and laws connected to ethics in my research context?
+​
+### Considerations
+* In all cases, your institution’s Data Protection Officer (DPO) is the person to refer to when considering ethical and legal aspects of data management.
+* When looking for recommendations and regulations, it is best to start from the local, that is, starting what is applicable within your discipline, and nationally. Then (if applicable), EU policies, directives and regulations are to be explored, as well as global recommendations (for example, from the UNESCO).
+* The list of resources given below does not attempt to be exhaustive: it tries to highlight the most relevant ones.
+​
+### Solutions
+* Use [domain](your_domain) and [national](national_resources) pages in the RDMkit as a starting point
+* Follow national legislation in terms of
+    * Data protection
+    * Conduct of health research, research on humans and animal studies
+    * Framework on research ethics
+    * Bio and national security
+* Consult national, regional, institutional Ethics Boards
+* Make use of guiding principles, recommendations, guidelines, ethical standards and declarations from professional associations and
+    * [WMA Declaration of Helsinki – Ethical Principles for Medical Research Involving Human Subjects](https://www.wma.net/policies-post/wma-declaration-of-helsinki-ethical-principles-for-medical-research-involving-human-subjects/)  
+    * For animal research: the [Three Rs principle](https://ec.europa.eu/health/scientific_committees/opinions_layman/en/non-human-primates/glossary/tuv/three-rs-principle.htm)  
+    * For data produced with Machine Learning (ML) techniques: the [recommendations from UNESCO](https://www.unesco.org/en/artificial-intelligence/recommendation-ethics), and the [DOME guidelines](https://dome-ml.org/), a product of the [ELIXIR ML focus group](https://elixir-europe.org/focus-groups/machine-learning)  
+* Resources and Policies from research infrastructures
+    * BBMRI ELSI Toolkit, Helpdesk and Services
+    * ELIXIR ELSI Policy [ELIXIR Ethics Policy](https://zenodo.org/record/557027#.ZGtrzpFBxD8)  
+* Check existing EU policies, directives and regulations
+    * The [General Data Protection Regulation](https://gdprinfo.eu/)
+    * The European [Clinical Trials Regulation](https://www.ema.europa.eu/en/human-regulatory/research-development/clinical-trials/clinical-trials-regulation)
+
```

Comment:
If you want to go beyond EU - to the US for example, this might be useful - https://www.hopkinsmedicine.org/institutional_review_board/hipaa_research/faq_research.html